static/js/client.js

```diff
@@ -1078,28 +1078,43 @@ var app = {}, browserSettings = {}, browserStorage = $.localStorage;
 
     }
 
-    function calcBGByTime(time) {
+    function scaledTreatmentBG(treatment) {
+
+      function calcBGByTime(time) {
         var closeBGs = data.filter(function(d) {
-            if (!d.y) {
-                return false;
-            } else {
-                return Math.abs((new Date(d.date)).getTime() - time) <= SIX_MINS_IN_MS;
-            }
+          if (!d.y) {
+            return false;
+          } else {
+            return Math.abs((new Date(d.date)).getTime() - time) <= SIX_MINS_IN_MS;
+          }
         });
 
         var totalBG = 0;
         closeBGs.forEach(function(d) {
-            totalBG += d.y;
+          totalBG += d.y;
```

Comment:
This needs to be totalBG += Number(d.y); for it to work. The values are strings and mg/dl calculations fail without the parsing.